webview/window.py

```diff
@@ -248,6 +249,13 @@ def toggle_fullscreen(self):
         """
         self.gui.toggle_fullscreen(self.uid)
 
+    @_shown_call
+    def toggle_topmost(self):
+        """
+        Toggle topmost mode
+        """
+        self.gui.toggle_topmost(self.uid)
+
```

Comment:
Though there is a `toggle_fullscreen`, I am not convinced about introducing more `toggle_` or `set_` functions. We have had a similar dicussion with x, y, width and height attributes and it was decided to implemented those as class properties. Maybe this one could be implemented in a similar fashion, ie `window.on_top = True`.
Opinions?